tests/test_auth.py

```diff
@@ -4,6 +4,8 @@
 import pytest
 import requests
 
+from up42.utils import get_up42_py_version
```

Comment:
`get_up42_py_version` depends on the package to be installed in Venv, otherwise it will fail with PackageNotFound. So we need to mock it.
That's why I'd add to conftest
```python
@pytest.fixture(autouse=True)
def mock_up42_py_version():
  with patch("importlib.metadata.version", return_value="test-version"):
    yield
```
And no changes are needed here, sorry for confusion